influxd.yaml

```diff
@@ -61,6 +67,76 @@ subpackages:
         - bash
         - yq
 
+  - name: ${{package.name}}-bitnami-compat
+    description: "compat package with bitnami/influxdb image"
+    dependencies:
+      runtime:
+        - bash
+        - coreutils
+        - curl
+        - findutils
+        - influxd
+        - influx
+        - grep
+        - libgcc
+        - procps
+        - sed
+        - wait-for-port # used in /opt/bitnami/scripts/libinfluxdb.sh
+    pipeline:
+      - uses: bitnami/compat
+        with:
+          image: influxdb
+          version-path: ${{vars.major-version}}/debian-12
+      - runs: |
+          mkdir -p ${{targets.subpkgdir}}/opt/bitnami/influxdb/bin
+          ln -sf /usr/bin/influxd ${{targets.subpkgdir}}/opt/bitnami/influxdb/bin/influxd
+          ln -sf /usr/bin/influx ${{targets.subpkgdir}}/opt/bitnami/influxdb/bin/influx
+
+          # copy etc to influxdb/etc.default
+          mkdir -p ${{targets.subpkgdir}}/opt/bitnami/influxdb/etc.default
+          cp -pr ${{targets.destdir}}/etc/* ${{targets.subpkgdir}}/opt/bitnami/influxdb/etc.default/
```

Comment:
NOT BLOCKING, FYI: `-p` preserving the attributes might be an issue during image testing since new owner will be `bitnami` user, so not sure about this. I'd use `install -m755` here. Or symlink that folder.